core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -531,26 +533,28 @@ private static Properties loadClientProperties(List<ConfigProperty> properties,
   }
 
   /**
-   * Loads the cluster level configuration from the get configuration response, and merges it with
-   * the existing configuration.
+   * Loads the cluster level configuration from the get configuration response,
+   * filters out the configuration for certain scope,
```

Comment:
nit: is there a reason to line break on the commas?

Reply:
Changed to no break, thanks